### PR TITLE
Add Stages.md, revise Working Mode to include

### PR DESCRIPTION
**Pull request by cwilso**
This moves the proposal for Stages from https://github.com/whatwg/meta/issues/290 to a Markdown file in SG, and revises the Working Mode document to include it.  Note that I tweaked some language slightly, and replaced a "stages review needed" label mention with our current "agenda+" label.

### CODE REVIEW

File: Working Mode.md
Comment (annevk) [inline Working Mode.md:112]:
```suggestion
Additionally, those maintaining such proposals should try to involve the WHATWG community, for example by filing an issue on the standard they anticipate eventually becoming a part of. We also have the optional [Stages](Stages.md) process to help gather attention on proposals as they progress.
```

**Review by annevk — COMMENTED** [review]
Thanks for putting this up!

For this to become part of our set of policy documents, it needs to read like one. At the point where something is policy, it's already agreed that we need to solve the problem and it's also no longer a proposal.

I think that mainly means that a lot of text here can be removed, though perhaps we want to keep some of it in some fashion as an appendix?

I think I'd also like to explicitly invite @whatwg/editors to review this. They are always welcome to review SG PRs, but this one in particular will have some impact on them.

File: Stages.md
Comment (jeremyroman) [inline Stages.md:28]:
```suggestion
    * The feature champion should bring the proposal to the WHATWG to advance to the next stage by setting an “Agenda+” label on the tracking issue, and showing up to (or ensuring someone will show up to) the next triage meeting to discuss. Advancing to a new stage requires support for the decision by at least two implementers (via their browser engine representatives), and there should not be any strong implementer objections ([per the working mode](https://whatwg.org/working-mode#additions)). This support can also be gathered in any public manner, e.g. GitHub issue comments, triage meetings, etc. 
```

File: Stages.md
Comment (jeremyroman) [inline Stages.md:17]: Should that process still be called the "current" process once this one is also current?

File: Stages.md
Comment (jeremyroman) [inline Stages.md:76]: These stray backslashes confused me. Are they intended to be there? It looks like they were intended to escape newlines in some different document format.

**Review by jeremyroman — COMMENTED** [review]
Just some trivial nits.

File: Stages.md
Comment (cwilso) [reply Stages.md]: Yeesh, yes, the bullets went haywire.  Fixed.

File: Stages.md
Comment (cwilso) [reply Stages.md:17]: Good catch, restructured.

Comment (jeremyroman) [reply Stages.md:17]: I don't see the change corresponding to this comment; have I missed it?

**Review by jeremyroman — COMMENTED** [review]
Makes sense to me; thank you for tolerating my nitpicking. Others may have more substantive feedback. 😄

File: Stages.md
Comment (cwilso) [reply Stages.md]: Gaaah, no, sorry, my fault; I'd committed but not pushed a change that had this wrapped in it.  It's there now:

* _Implementation interest_: a direct signal from a browser engine that a feature is on the roadmap to implement, as defined in [working mode](https://whatwg.org/working-mode#additions)].

**Review by jeremyroman — APPROVED** [review]

File: Stages.md
Comment (domenic) [inline Stages.md:1]: Not sure if a policy needs a subtitle in this way; it would feel a bit out of place among the others on https://whatwg.org/policies I think. Maybe just `# Stages` would work.

File: Stages.md
Comment (domenic) [inline Stages.md:3]:
```suggestion
The WHATWG's "document reality" approach has been effective for nailing down [fundamental parts of the platform](https://spec.whatwg.org/) and improving interoperability and developer satisfaction. However, our process can sometimes be daunting for newcomers, who don't know how to reliably get implementer feedback or editor time commitment. This can discourage contributions, and it can lead to frustration when implementer/editor feedback arrives late in the process after the contributor feels "close to done".
```

File: Stages.md
Comment (domenic) [inline Stages.md:7]:
```suggestion
Stage signals are also useful to the broader community, helping developers monitor the proposals that are moving through the various stages.  By explicitly signaling a proposal's progress, including implementer involvement, the community has a better idea of what is going on in the WHATWG.

This tool is generally used for medium-to-large new features; it's not expected that explicit stage progression will be used for every feature.
```

File: Stages.md
Comment (domenic) [inline Stages.md:11]:
```suggestion
This document formalizes some signal checkpoints for new features in existing WHATWG Living Standards. These checkpoints are modeled loosely on the [TC39 process](https://tc39.es/process-document/), which uses the concept of _stages_. Each subsequent stage implies a larger degree of consensus from the community, willingness to engage and/or implement and ship the feature in browser engines, and signals progress to web developers.  These checkpoints are not required for all features; this is simply a way to ensure that attention is appropriately drawn to incubations that have grown community support behind them.
```

File: Stages.md
Comment (domenic) [inline Stages.md:15]:
```suggestion
* _Stage_: each successive stage is closer to a landed spec and interoperable implementation.  These stages are tracked on the issues in the relevant WHATWG GitHub repository, using labels ("stage 0", "stage 1", etc.)
```

File: Stages.md
Comment (domenic) [inline Stages.md:17]:
```suggestion
* _Implementation interest_: a direct signal from a browser engine that a feature is on the roadmap to implement, as defined in [working mode](https://whatwg.org/working-mode#additions).
```

File: Stages.md
Comment (domenic) [inline Stages.md:13]:
```suggestion
## Terminology
```

File: Stages.md
Comment (domenic) [inline Stages.md:21]:
```suggestion
## Process
```

File: Stages.md
Comment (domenic) [inline Stages.md:15]: Consider using `<dfn>` instead of `_`

File: Stages.md
Comment (domenic) [inline Stages.md:28]: The first sentence about agenda+ and triage meetings seems to contradict the last sentence about gathering support in any manner. Maybe it can be deleted?

File: Stages.md
Comment (domenic) [inline Stages.md:43]: Make these th instead of td, and probably it'd be good to introduce thead and tbodys.

**Review by domenic — APPROVED** [review]
Looks great! I just found a few wording and formatting nits that can make it fit in better as a policy.

File: Stages.md
Comment (cwilso) [reply Stages.md:30]: Although I agree the last sentence is not entirely necessary, it's not contradictory - the first sentence is about "how do you bring the proposal to the group", the last sentence is "how do you demonstrate support from implementers".

File: Stages.md
Comment (cwilso) [reply Stages.md]: Fixed in https://github.com/whatwg/sg/pull/222/commits/dd195992df4f01dbd374454f6642065ec8a2bbe9.

File: Stages.md
Comment (cwilso) [reply Stages.md]: fixed in https://github.com/whatwg/sg/pull/222/commits/dd195992df4f01dbd374454f6642065ec8a2bbe9.

File: Stages.md
Comment (cwilso) [reply Stages.md]: Done in https://github.com/whatwg/sg/pull/222/commits/dd195992df4f01dbd374454f6642065ec8a2bbe9.

**Comment by cwilso** [comment]
Thanks for all the feedback, @domenic !  Addressed (nearly) everything.

**Review by tantek — APPROVED** [review]
This overall LGTM. I have some purely editorial suggestions for punctuation/grammar but they can wait until this is landed.

**Comment by cwilso** [comment]
2 approvals, open for several weeks - merging.

**Pull request merged**
